packages/playground/src/components/repo-clone/clone-repo-dialog.tsx

```diff
@@ -2,18 +2,23 @@ import { CloneRepoForm } from './clone-repo-form'
 import { Tabs, TabsList, TabsTrigger, Popover, PopoverContent, PopoverTrigger, Button, Text } from '@harnessio/canary'
 import React, { useState } from 'react'
 
-export const CloneRepoDialog = () => {
+export const CloneRepoDialog = ({
+  httpsUrl,
+  sshUrl,
+  handleCreateToken
+}: {
+  sshUrl: string
+  httpsUrl: string
+  handleCreateToken: () => void
```

Comment:
let's pull this into an exported interface `CloneRepoDialogProps`. helps with debugging/autocomplete etc.